.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Your github action
+
+on:
+  push:
+    branches:
+      # NOTE: You may want to limit the trigger branch to be "main" or "master" etc.
+      - '*'
```

Comment:
yeah we prob just want this one on `main` right?  otherwise every PR will clobber our docs

Reply:
Also the plan. I did this for testing github action on my fork.